applications/GeoMechanicsApplication/tests/cpp_tests/test_calculate_incremental_displacement_process.cpp

```diff
@@ -0,0 +1,63 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+#include "custom_processes/calculate_incremental_displacement_process.h"
+#include "geo_mechanics_application_variables.h"
+#include "geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+using namespace Kratos;
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateIncrementalDisplacementProcess, KratosGeoMechanicsFastSuite)
+{
+    // set up model part
+    Model model        = Model();
```

Comment:
The following is more compact and equivalent:
```suggestion
    Model model;
```